SECEdgar/filings/base.py

```diff
@@ -18,17 +18,16 @@ class Filing(_EDGARBase):
 
     .. versionadded:: 0.1.5
     """
-    _VALID_FILING_TYPES = ["10q", "10-q", "10k",
-                           "10-k", "8k", "8-k",
-                           "13f", "13-f", "4", "sd"]
+    _VALID_FILING_TYPES = ["10-q", "10-k", "8-k", "13-f", "4", "sd"]
```

Comment:
I originally added the 10q, 10k, 8k, and 13f options because it seemed to be more lenient. What are your thoughts on this?

Reply:
My problem is that if you request the URL without the hyphen, for example [like this](https://www.sec.gov/cgi-bin/browse-edgar?action=getcompany&CIK=0000320193&type=10k&dateb=&owner=exclude&output=xml&count=100), no search result are returned. This is why I have removed the abbreviated file type.

Maybe you can pre-process the document type and manually insert the hyphen if it is missing.

On the other hand, I am not sure why we can't download file types other than the ones listed too? There are some explanations [https://www.sec.gov/oiea/Article/edgarguide.html](here)

Reply:
I think you are right. We should not include the options without hyphens if the hyphens are needed.

Reply:
I was thinking of having enum for this reason as underneath changes will not affect the overall API but it is okay for now.